charts_plugin/src/main/java/com/futureworkshops/mobileworkflow/plugin/charts/view/ItemProvider.kt

```diff
@@ -87,9 +87,10 @@ sealed class ItemProvider {
                             listItemId = it.id,
                             chartType = it.chartType,
                             title = localizationService.getTranslation(it.title),
+                            text = it.text,
```

Comment:
@jmcarcedo Shouldn't this text also be translated?

Reply:
@igorcferreira you are correct, changed in [fca2162](https://github.com/FutureWorkshops/MWChartsPlugin-Android/pull/13/commits/fca21627e9541c4774df647a3f95c467585e30d1)

Reply:
Yeah sorry for that I got distracted. Changed in [e71a216](https://github.com/FutureWorkshops/MWChartsPlugin-Android/pull/13/commits/e71a2169b8755bd0858f59bd226a282ae674d8eb)